### DISCUSSION TITLE
How to I push updates?

### DISCUSSION THREAD

**Discussion by CxGrizzy** [Frederisk/Wikitext-Extension-Gadget#8] [Q&A] [answered]
Hello! I'm not sure how to push updates over? Do I just edit them using VS Code and then copy and paste the code into the Wiki or is it possible to push an update from VS Code? 

Thank you sorry if you answered it elsewhere.

**Comment by Frederisk** [comment, marked as answer]
This is actually a question about [Wikitext-VSCode-Extension](https://github.com/Frederisk/Wikitext-VSCode-Extension).

Just read its README.md:

> - You can modify the website content by logging in directly in VSCode! Enter the settings page, search Wikitext and find the username and password items, and fill in them. Then press F1 in the text editor, select Post your page to the website!
> - Obtain the Wikitext source code directly in VSCode according to the page name without opening the web page. Press F1 and select Pull page to edit to use this function.

**Reply by CxGrizzy** [reply]
Thank you! I just installed it from VS code and never checked sorry about that!
